### DISCUSSION TITLE
پیشنهاد برای پارسی دیت

### DISCUSSION THREAD

**Discussion by bihovi584om** [wordpress-parsi/wp-parsidate#175] [Ideas]
سلام
سپاس از تیم وردپرس پارسی و مشارکت کننده ها پیشنهادی برای این افزونه دارم؛
نمیشه تبلیغ توی افزونه قرار بدید یا لینک سایت قرار بدید یا توی ویجت صفحه اصلی اخبار سایت مرتبط قرار بدید یا روش درامدزای دیگه پیدا کنید ولی در عوض سریعتر اپدیت بدید و یه تیم تمام یا نیمه وقت برای این افزونه قرار بدید؟ چون واقعا تقویم فارسی برای جامعه فارسی زبان وردپرس فارسی نیازه ضروری هستش.
مرسی ازشما

بازخورد شما عزیزان رو میشنوم؛

**Comment by bihovi584om** [comment]
یا نمیشه در هسته وردپرس تقویم شمسی رو هم اضافه کنید؟
باهمکاری تیم خارجی وردپرس کدهای تقویم شمسی رو در هسته وردپرس اضافه کنید
میشه آیا؟؟

**Comment by man4toman** [comment]
سلام 
ممنونم بابت پیشنهادتون. مشارکت در توسعه این افزونه کلا براساس قوانین دنیای کدباز بوده و هست ولی بابت توسعه بهتر و منظم‌تر برنامه‌هایی داریم که به زودی اجرایی میشن.

قبلتر پیشنهاد انتقال تاریخ شمسی به هسته رو داده بودیم که به جایی نرسید.
